composition/src/v1/utils/constants.ts

```diff
@@ -657,6 +659,14 @@ export const OVERRIDE_DEFINITION: DirectiveDefinitionNode = {
   repeatable: false,
 };
 
+// directive @openfed__protected repeatable on FIELD_DEFINITION | OBJECT_DEFINITION
+export const PROTECTED_DEFINITION: DirectiveDefinitionNode = {
+  kind: Kind.DIRECTIVE_DEFINITION,
+  locations: stringArrayToNameNodeArray([FIELD_DEFINITION_UPPER, OBJECT_UPPER]),
+  name: stringToNameNode(PROTECTED),
+  repeatable: true,
```

Comment:
Is there any reason why it should be repeatable?

Reply:
If it's not repeatable, you can't add it to a type and an extension.